src/NServiceBus.AcceptanceTests/EndpointTemplates/DefaultServer.cs

```diff
@@ -61,6 +61,23 @@ public DefaultServer(List<Type> typesToInclude)
             await builder.DefinePersistence(settings);
 
             builder.GetSettings().SetDefault("ScaleOut.UseSingleBrokerQueue", true);
+            builder.NotifyOnFailedMessage(failedMessage =>
```

Comment:
@timbussmann the problem is we are back door-ing asserting exceptions via the notifications.